.github/workflows/build-push-image.yml

```diff
@@ -0,0 +1,33 @@
+name: Tutor Build Action
+
+on:
+  push:
+    branches:
+      - 'pearson-release/olive.main'
+  pull_request:
```

Comment:
@Squirrel18 Do we want to trigger the action to build the image every time a change is made to a PR or only when it is merged?

Reply:
It should be triggered every time we update the main branch, which means only when a PR is merged. @sergivalero20 @alexjmpb

Reply:
Ok ok, thanks @alexjmpb @Squirrel18